src/reducers/__snapshots__/node.reducer.test.js.snap

```diff
@@ -556,6 +556,74 @@ Object {
 }
 `;
 
+exports[`Check node reducer FLOWDESIGNER_NODE_SET_TYPE update node type 1`] = `
+Object {
+  "childrens": Object {},
+  "in": Object {},
+  "links": Object {},
+  "nodeTypes": Object {},
+  "nodes": Object {
+    "id1": Object {
+      "data": Object {
+        "type": "test",
+      },
+      "getNodeType": [Function],
+      "getPosition": [Function],
+      "getSize": [Function],
+      "graphicalAttributes": Object {
+        "description": "",
+        "label": "",
+        "nodeSize": Object {
+          "height": undefined,
+          "width": undefined,
+        },
+        "nodeType": undefined,
```

Comment:
so here it's undefined

Reply:
because what is tested is the node.type and not the node.graphicalAttributes.type, what is tested is on line 88